src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -571,6 +571,69 @@ static size_t uct_cuda_copy_md_get_total_device_mem(CUdevice cuda_device)
     return 1; /* return 1 byte to avoid division by zero */
 }
 
+/**
+ * Get information on memory allocations.
+ *
+ * @param [in]  address        Pointer to the memory allocation to query
+ * @param [in]  length         Size of the allocation
+ * @param [in]  ctx            CUDA context on which a pointer was allocated.
+ *                             NULL in case of VMM
+ * @param [out] base_address_p Returned base address
+ * @param [out] alloc_length_p Returned size of the memory allocation
+ *
+ * @return Error code as defined by @ref ucs_status_t.
+ */
+static ucs_status_t
+uct_cuda_copy_md_get_address_range(const void *address, size_t length,
+                                   CUcontext ctx, void **base_address_p,
+                                   size_t *alloc_length_p)
+{
+    ucs_log_level_t log_level = (ctx != NULL) ? UCS_LOG_LEVEL_DEBUG :
+                                                UCS_LOG_LEVEL_ERROR;
```

Comment:
why?

Reply:
It should be vice versa. We should only print `cuMemGetAddressRange` error when we set the right context.